src/main/java/com/wavefront/opentracing/reporting/WavefrontSpanReporter.java

```diff
@@ -37,7 +39,7 @@ private static String getDefaultSource() {
       try {
         return InetAddress.getLocalHost().getHostName();
       } catch (UnknownHostException ex) {
-        return "wavefront-tracer-reporter";
+        return DEFAULT_SOURCE;
```

Comment:
Should the default source be `wavefront-tracer-reporter` or `wavefront-tracer` ?

Reply:
Let's go with `wavefront-tracer ` since that's shorter.